java/cdk/src/main/java/sleeper/cdk/stack/DashboardStack.java

```diff
@@ -47,26 +46,28 @@
 
 import static sleeper.configuration.properties.instance.CommonProperty.ID;
 import static sleeper.configuration.properties.instance.CommonProperty.METRICS_NAMESPACE;
+import static sleeper.configuration.properties.instance.CommonProperty.REGION;
 import static sleeper.configuration.properties.instance.DashboardProperty.DASHBOARD_TIME_WINDOW_MINUTES;
 
-@SuppressFBWarnings("MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR")
 public class DashboardStack extends NestedStack {
-    private final IngestStack ingestStack;
-    private final CompactionStack compactionStack;
-    private final PartitionSplittingStack partitionSplittingStack;
     private final String instanceId;
     private final List<String> tableNames;
     private final String metricsNamespace;
     private final Duration window;
     private final Dashboard dashboard;
+    private final List<IMetric> errorMetrics;
```

Comment:
It might be a bit clearer if we pass the error metrics list around rather than setting it as a field. It looks like it's only used in one method?